common/dynamicconfig/constants.go

```diff
@@ -3688,6 +3714,11 @@ var BoolKeys = map[BoolKey]DynamicBool{
 		Description:  "Lockdown defines if we want to allow failovers of domains to this cluster",
 		DefaultValue: false,
 	},
+	EnablePendingActivityValidation: DynamicBool{
+		KeyName:      "limit.pendingActivityCount.enabled",
+		Description:  "",
```

Comment:
Could you add a description here as well?

Reply:
added.